Dockerfile

```diff
@@ -1,71 +1,74 @@
 # syntax=docker/dockerfile:1
 
-# Make sure to keep in sync with the version in rust-toolchain.toml
-ARG RUST_VERSION=1.71
-
+# Make sure to keep RUST_VERSION in sync with the version in rust-toolchain.toml
+ARG BASE_IMAGE="bullseye"
+ARG RUST_VERSION=1.71.1
```

Comment:
would it be possible to query the `rust-toolchain.toml` and pass that value in?

Reply:
Unfortunately we can't do that *here*. We could do that at runtime, when we call `docker build`, but we would still need to hardocde a default value here.